### ISSUE TITLE
User story: page should be accessible for those using screen-readers

### ISSUE THREAD

**Issue by joe-dev-public** [sonianb/fac-to-do-list#34]
- e.g. make sure controls etc. have labels that make the function clear to a screen-reader
